src/styles/blocks/page.scss

```
@@ -0,0 +1,16 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  &__container {
+    display: flex;
+    width: calc(4 * #{$card-width} + 3 * #{$right-card-margin});
+    flex-wrap: wrap;
+    align-items: center;
+    margin: 0 auto;
+    box-sizing: border-box;
+    padding-top: 51px;
```

```suggestion
    padding: 51px 41px;
```
You should also add horizontal padding for your catalog